packages/dara-core/dara/core/interactivity/filtering.py

```diff
@@ -38,6 +38,7 @@ class Pagination(BaseModel):
     offset: Optional[int] = None
     limit: Optional[int] = None
     orderBy: Optional[str] = None
+    index: Optional[str] = None
```

Comment:
Let's update the docstring to explain how `index` affects the returned results

Reply:
Good point, updated